### PR TITLE
CI: Switch from Travis CI to GitHub Actions, use Docker for CrateDB

### PR DESCRIPTION
**Pull request by amotl**
In order to set the stage for #16, the patch removes outdated mechanisms to run CrateDB, and uses Docker instead. It also switches CI from Travis CI to GitHub Actions.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle install
+
+      - name: Run tests
+        run: bundle exec rspec
+
+      #- name: Run rubocop
+      #  run: bundle exec rubocop
```
Comment (hammerhead) [inline .github/workflows/tests.yml:50]: Rubocop isn't set up as a dependency right now, so remove it from here?

File: DEVELOP.md [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+```shell
+docker run --rm -it --publish=44200:4200 crate:5.1.1 \
+  -Cdiscovery.type=single-node \
+  -Ccluster.routing.allocation.disk.threshold_enabled=false
+```
+
+Install project and invoke test suite.
+```shell
+bundle install
+bundle exec rspec
```
Comment (hammerhead) [inline DEVELOP.md:13]: I was trying to run the tests locally on this branch, but it didn't work:
`/Users/user/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/activesupport-4.1.16/lib/active_support/core_ext/object/duplicable.rb:82:in <class:BigDecimal>: undefined method new for BigDecimal:Class (NoMethodError)`

Will this be fixed with #16?

File: .github/workflows/tests.yml (hunk 1 above)
Comment (amotl) [reply .github/workflows/tests.yml]: Thanks. Fixed.

File: DEVELOP.md (hunk 2 above)
Comment (amotl) [reply DEVELOP.md:13]: Yes, I've made it pretty far. I think making it work completely, and successfully, is almost around the corner. If you like, and if you can spare some minutes, I will be happy about any help on #16.

**Review by hammerhead — APPROVED** [review]

**Pull request merged**